docs/application/design/tv/patterns/screen-layout.md

```diff
@@ -2,6 +2,50 @@
 
 The TV screen is divided into the following 3 parts to help the user achieve desired goals:
 
--   [Header area](screen-layout/header-area.md)
--   [Content area](screen-layout/content-area.md)
--   [Footer area](screen-layout/footer-area.md)
+-   [Header area](#header-area)
+-   [Content area](#content-area)
+-   [Footer area](#footer-area)
+
+## Header Area
+
+A header area is the top portion of the view. It provides information about the title of the current content and screen.
+
+The following guidelines apply to the header area:
+
+-   The header area can have tabs and a title bar.
+-   The title bar is displayed above the tabs.
+
+![header](media/pt_08_footer_area_re-850x478.png)<br>
```

Comment:
FYI : This pt_08_footer_area_re-850x478.png image appears 3 times in this one page to explain Header, Content, and Footer area. It looks like this duplication can be removed.